### DISCUSSION TITLE
Best configuration for rtl_wmbus option

### DISCUSSION THREAD

**Discussion by alvaromemo** [wmbusmeters/wmbusmeters#1016] [General]
Im using a RTL dongle to detect 433Mhz water meters using this options.

`wmbusmeters rtlwmbus:434.475M:t1,c1 MyWater izar '*' NOKEY`

It works perfectly, but i want to know how to:

- Use more Gain or set to Auto (AGC)
- Use a different sample frequency; i mean, for wmbus i know it must be 1200Khz, but in case it is different, how i set it?
- Which are the best settings to have more sensitivity and get more telegrams. Right now i'm using a 4dbi dipole antenna. 
Thank you.

**Comment by weetmuts** [comment]
You can specify your own rtlwmbus  command line like this:

`wmbusmeters 'rtlwmbus:CMD(/usr/bin/rtl_sdr -f 868.95M -s 1600000 - 2>/dev/null | /usr/bin/rtl_wmbus -p s)' MyWater izar ANYID NOKEY`

You can run this command outside of wmbusmeters if you just want to check for reception of telegrams.

You can modify the sample frequency as long as it is a multiple of 800KHz and you specify -d to rtl_wmbus.
Check the README at https://github.com/xaelsouth/rtl-wmbus

You have to read up on the rtl_sdr settings for gain etc. I am not an expert on these unfortunately.

Here is the current available information on antennas and long range:
https://github.com/wmbusmeters/wmbusmeters-wiki/blob/master/ANTENNA.md
https://github.com/wmbusmeters/wmbusmeters-wiki/

**Comment by alvaromemo** [comment]
Ok, can you explain why 800Khz?, which are the implication of the sampling rate?.

**Reply by weetmuts** [reply]
No sorry. But it is probably due to how the decoding is implemented in rtl_wmbus. The sample rate was initially fixed, but now you can change it as long as you provide the correct -d  option to rtl_wmbus.
